guides/howtos/Schemas and changesets.md

```diff
@@ -0,0 +1,121 @@
+# Schemas and changesets
+
+We will take a look at the role schemas play when validating and casting data through changesets. As we will see, sometimes the best solution is not to completely avoid schemas, but break a large schema into smaller ones. Maybe one for reading data, another for writing. Maybe one for your database, another for your forms.
+
+## Schemas are mappers
+
+[The Ecto documentation](https://hexdocs.pm/ecto/Ecto.Schema.html) says:
```

Comment:
We should rewrite any links to Ecto itself as regular code links. For example, here we could say:

> The `Ecto.Schema` moduledoc says: